analytics/report/README.md

```diff
@@ -1 +1,95 @@
 # README about reporting module
+
```

Comment:
@nam-m @alchuu00 have you guys thought about putting the poetry and poetry lock under analytics folder instead of each having its own lock files? 
otherwise how would you resolve the dependencies when you run the program together.

Reply:
Right, thanks for pointing this out. Will discuss it with Nam.